src/processing_eeg_methods/data_loaders.py

```diff
@@ -191,13 +191,7 @@ def torres_dataset_loader(
     y = [0, 1, 2, 3, 4]
     y = np.repeat(y, 33, axis=0)
 
-    event_dict = {
-        "Arriba": 0,
-        "Abajo": 1,
-        "Izquierda": 2,
-        "Derecha": 3,
-        "Seleccionar": 4,
-    }
+    dataset_info["event_dict"]
     return x, y, event_dict
```

Comment:
Here too. Delete the line 194 and when returning, don't return the event_dict. It should only be `return x, y`. Similarly repeat with the other functions.